migrations/migrate-duplicate-emails.sql

```diff
@@ -0,0 +1,52 @@
+--------------------------------------------------------------------------------
+-- For each relation, find all records tied to a learn user that also have a
+-- github account with the same email, re-associate the records with the github
+-- account
+--------------------------------------------------------------------------------
+UPDATE membership
+  SET "user" = guser.id
+  FROM "user" luser
+  JOIN "user" guser
+    ON guser.email = luser.email
+  WHERE luser.id = membership.user
+    AND guser.plugin = 'github'
```

Comment:
Will this set their `plugin` to "github" but leave a Learn UID in `ident`? Is there a potential collision issue there?